.github/workflows/uninstall_install_sandbox.yaml

```diff
@@ -27,18 +30,29 @@ jobs:
           openshift_token: ${{ secrets.OPENSHIFT_TOKEN }}
           namespace: ${{ secrets.OPENSHIFT_NAMESPACE }}
       
-      - name: Uninstall Traction via Helm
-        continue-on-error: true
-        run: |
-          helm uninstall traction-sandbox -n ${{ secrets.OPENSHIFT_NAMESPACE }} --wait --timeout=20m
+      # - name: Uninstall Traction via Helm
+      #   run: |
+      #     helm uninstall traction-sandbox -n ${{ secrets.OPENSHIFT_NAMESPACE }} --wait --timeout=20m
 
-      - name: Remove Traction Openshift Objects
-        continue-on-error: true 
-        run: |
-          oc delete -n ${{ secrets.OPENSHIFT_NAMESPACE }} all,secret,pod,networkpolicy,configmap,pvc --selector "app.kubernetes.io/instance"=traction-sandbox
+      # - name: Remove Traction Openshift Objects
+      #   run: |
+      #     oc delete -n ${{ secrets.OPENSHIFT_NAMESPACE }} all,secret,pod,networkpolicy,configmap,pvc --selector "app.kubernetes.io/instance"=traction-sandbox
```

Comment:
Did you mean to comment out the `Uninstall Traction via Helm` and `Remove Traction Openshift Objects` steps?